packages/experimental/Link/src/Link.types.ts

```diff
@@ -0,0 +1,97 @@
+import { TextTokens, TextProps } from '@fluentui-react-native/text';
+import { IFocusable, IPressableState, IWithPressableEvents, IWithPressableOptions } from '@fluentui-react-native/interactive-hooks';
+import { ViewProps } from 'react-native';
+
+export const linkName = 'Link';
+
+export type ILinkState = IPressableState & {
+  /**
+   * Specifies whether the link has been visited.
+   * @default false
+   */
+  visited?: boolean;
+  /**
+   * Specifies whether the link is disabled.
+   * @default false
+   */
+  disabled?: boolean;
+  /**
+   * Specifies whether the link is inline.
+   * @default false
+   */
+  inline?: boolean;
+  /**
+   * Specifies whether the link is subtle.
+   * @default false
+   */
+  subtle?: boolean;
+};
+/**
+ * Link tokens, these are the internally configurable values for Link elements. In particular these
+ * drive decisions on how to build the styles
+ */
+export interface LinkTokens extends TextTokens {
+  hovered?: LinkTokens;
+  pressed?: LinkTokens;
+  focused?: LinkTokens;
+  visited?: LinkTokens;
+  disabled?: LinkTokens;
+  inline?: LinkTokens;
+  subtle?: LinkTokens;
+}
+
+export type LinkAppearance = 'default' | 'subtle';
+
+export interface LinkOptions {
+  /**
+   * The URL that is opened when the link is clicked.  This value supersedes the 'onPress' callback when both are present.
+   * @default undefined
+   */
+  url?: string;
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type IWithLinkOptions<T extends object> = LinkOptions & IWithPressableOptions<T>;
```

Comment:
What's the reason for having this as a separate type? It doesn't seem to be used anywhere else

Reply:
removed!